src/debug/components/ValidationResponseCard.tsx

```diff
@@ -109,22 +118,37 @@ export const ValidationResponseCard = ({
             style={{ display: "flex", flexDirection: "column", gap: "2 rem" }}
           >
             {renderSummary()}
-
-            <Button
-              title="Expand"
-              variant="tertiary"
-              size="sm"
-              onClick={() => setIsExpanded(!isExpanded)}
-              style={
-                {
-                  // ...styles.responsiveButtons,
-                  alignSelf: "flex-end",
-                  marginBottom: "1rem",
-                } as React.CSSProperties
-              }
+            <Box
+              gap="xs"
+              direction="row"
+              style={{ alignSelf: "flex-end", marginBottom: "1rem" }}
             >
-              {isExpanded ? "Hide " : "Show "} Details
-            </Button>
+              <Button
+                title="Expand"
+                variant="tertiary"
+                size="sm"
+                onClick={() => setIsExpanded(!isExpanded)}
+              >
+                {isExpanded ? "Hide " : "Show "} Details
+              </Button>
+              {txHash ? (
+                <Button
+                  title="Expand"
+                  variant="tertiary"
+                  size="sm"
+                  onClick={() =>
+                    window.open(
```

Comment:
@hugo-heer This should be local lab if the app is running locally, or link out to https://lab.stellar.org/ otherwise. If someone's running on testnet, they might not have the docker container running locally

Reply:
Also, can this be changed to a Link with href instead of Button with onClick?